README.md

```diff
@@ -29,16 +29,17 @@ This repo contains a (C++, python (experimental)) linter and auto formatter pack
 ## Installation
 
 ```bash
-cd $YOUR_REPO
-git submodule add git@github.com:ethz-asl/linter.git
-./linter/init-git-hooks.py
+git clone git@github.com:ethz-asl/linter.git
+cd linter
+echo ". $(realpath setup_linter.sh)" >> ~/.bashrc  # Or the matching file for
```

Comment:
can we make this a simple script, so the installation is done by simply calling ./setup.sh or sth like this?

Reply:
It's actually not that easy to do if you want to support different shells, not everyone is using bash :) Anyway it's only one command to run